Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedOperation.swift

```diff
@@ -18,6 +18,11 @@ enum UnifiedOperationResult {
     case rootDocumentArray([BSONDocument])
     case none
 
+    /// Some tests for unacknowledged writes assume that the result is non-nil with an "acknowledged" field
```

Comment:
I considered adding special matching logic for when the "acknowledged" field is expected, but that seemed messier. I also considered just returning empty documents in those cases, since that would also satisfy the "unset or matches" operator, but it seemed a little nicer to use a document like this so it's clear what the return value is indicating.